### ISSUE TITLE
Not possible to disable

### ISSUE THREAD

**Issue by fabianwohlfart** [nuxt/devtools#84]
Hi guys,

so i just tested these devtools (with Node 18), but now I cannot disable it anymore.
Even though it says

`Nuxt Devtools is not enabled for this project.`

I always get

`✔ Nuxt Devtools is enabled v0.1.2 (experimental)`

Also if i switch to another project which runs on Node 16, I always get Node 18 errors as this was the version i initially installed the devtools.

`The request url "/Users/…/.nvm/versions/node/v18.10.0/lib/node_modules/@nuxt/devtools/dist/runtime/plugins/devtools.client.mjs" is outside of Vite serving allow list.`

Therefor I cannot use any of my other nuxt3 projects anymore.

`.nuxtrc` is not existent / empty. Is it mandatory?

How to fix this?

**Comment by antfu** [comment]
It's `~/.nuxtrc` under your home directory - Nuxt DevTools creates it to be serve as a global module, can you check again?

**Comment by fabianwohlfart** [comment]
```
telemetry.consent=1
telemetry.enabled=true
telemetry.seed=5a121daa084eedb3
modules.0=/Users/…/.nvm/versions/node/v18.10.0/lib/node_modules/@nuxt/devtools/module.cjs
devtoolsGlobal.projects=[]
```

**Comment by antfu** [comment]
If you are switching node versions often, maybe try to remove the last two items in your `.nuxtrc` and use devtools as a module: https://github.com/nuxt/devtools#install-manually

**Comment by fabianwohlfart** [comment]
Alright, now I understand that this is the way to go. First time I wanted to use the devtools I simply did everything within the install-section of the docs. Only later I understood that it must not be installed globally.

On the other side the Node 18 / Node 16 error was not the main problem.
The main problem was that even if I did `npx nuxi@latest devtools disable` the current project still tries to enable the devtools. Cleaning the ~/.nuxtrc also helps here.
